src/main/scala/org/allenai/wikitables/TestWikiTablesCli.scala

```diff
@@ -131,6 +142,44 @@ class TestWikiTablesCli extends AbstractCli() {
 
     denotations
   }
+
+  def answerSingleQuestion(modelFilename: String, options: OptionSet): Map[String, List[(Value, Double)]] = {
+    // TODO: Do not load model for each question. Different Cli?
```

Comment:
You're going to want to start a service that loads the model then listens for requests.  That can be done later, though.